src/app/header/header.component.ts

```diff
@@ -1,7 +1,7 @@
 import { MenuItem } from './../models/menu-item';
 import { ProfileService } from './../profile/profile.service';
 import { Component, OnInit } from '@angular/core';
-import { Router, NavigationEnd } from '@angular/router';
+import { Router, NavigationStart } from '@angular/router';
```

Comment:
how did you even figure out to use nav start instead of nav end? I missed this one.